playwright/fabric-custom.test.ts

```diff
@@ -0,0 +1,56 @@
+import { expect, test } from '@playwright/test';
+
+const viewport = { width: 1600, height: 1000 };
+
+const widths = ['360', '740', '980', '1300', '100%'];
+
+test.describe('Fabric Custom visual regression testing', () => {
+	test('Get reference screenshots', async ({ page }) => {
+		await page.setViewportSize(viewport);
+
+		await page.goto(
+			'https://guardian.github.io/commercial-templates/ssr/fabric-custom/',
+			{
+				waitUntil: 'networkidle',
+			},
+		);
+
+		for (const breakpoint of widths) {
```

Comment:
The viewport needs to be a minimum width to view some of the template previews hence why we set that at the start. The only relation between that and the widths/breakpoints is that it needs to be bigger than 1300px, the widest preview size.

The breakpoints correspond to the different sizes of template previews we have on one page. So we load the page, and then iterate through the width values because we can use these as selectors for the different iframes.

So the 1300 value is used to select the iframe containing the 1300 sized template preview. So we're iterating through these different iframes:
<img width="733" alt="Screenshot 2024-07-08 at 16 32 35" src="https://github.com/guardian/commercial-templates/assets/108270776/d3cffce6-9183-41b2-9e61-1a5d4f3ab209">

Reply:
I see thanks for explaining!